android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -254,15 +254,22 @@ public LiveData<List<TransactionInfo>> getAllTransactions() {
 
     public List<CryptoAccountTypeInfo> getSupportedCryptoAccountTypes() {
         List<CryptoAccountTypeInfo> cryptoAccountTypeInfos = new ArrayList<>();
+        cryptoAccountTypeInfos.add(new CryptoAccountTypeInfo(
+                mContext.getString(R.string.brave_wallet_create_account_ethereum_description),
+                mContext.getString(R.string.wallet_eth_name), CoinType.ETH, R.drawable.eth));
+
         if (isSolanaEnabled()) {
             cryptoAccountTypeInfos.add(new CryptoAccountTypeInfo(
                     mContext.getString(R.string.brave_wallet_create_account_solana_description),
                     mContext.getString(R.string.wallet_sol_name), CoinType.SOL,
                     R.drawable.ic_sol_asset_icon));
         }
-        cryptoAccountTypeInfos.add(new CryptoAccountTypeInfo(
-                mContext.getString(R.string.brave_wallet_create_account_ethereum_description),
-                mContext.getString(R.string.wallet_eth_name), CoinType.ETH, R.drawable.eth));
+        if (isFilecoinEnabled()) {
+            cryptoAccountTypeInfos.add(new CryptoAccountTypeInfo(
+                    mContext.getString(R.string.brave_wallet_create_account_filecoin_description),
+                    mContext.getString(R.string.wallet_fil_name), CoinType.FIL,
+                    R.drawable.ic_fil_asset_icon));
+        }
```

Comment:
I've included the Filecoin type, when enabled, but also modified the order of the list to show Ethereum in the first place like on Desktop.
Now the order will be the same for both platforms: Ethereum, Solana (if enabled), and Filecoin (if enabled).